### PR TITLE
Fix dictionary changed size during iteration while converting levels map

### PR DESCRIPTION
**Pull request by LMByrne**
+ deal with string digit entry_ids leading to duplicate entries. 
+ default for premium sub count

### CODE REVIEW

File: disco/bot/bot.py [hunk 1]
```diff
@@ -189,7 +189,8 @@ def __init__(self, client, config=None):
         # Convert our configured mapping of entities to levels into something
         #  we can actually use. This ensures IDs are converted properly, and maps
         #  any level names (e.g. `role_id: admin`) map to their numerical values.
-        for entity_id, level in six.iteritems(self.config.levels):
+        for entity_id, level in six.iteritems(self.config.levels.copy()):
```
Comment (b1naryth1ef) [inline disco/bot/bot.py]:
```suggestion
        for entity_id, level in list(six.iteritems(self.config.levels)):
```

**Comment by b1naryth1ef** [comment]
I don't understand what (apart from the dictionary runtime error) this solves. Can you elaborate?

**Comment by LMByrne** [comment]
For entries where the key was originally a digit string in the permissions map, you'd end up with 2 entries (one with a string key and one with an int key) if you don't delete the original entry.

**Comment by b1naryth1ef** [comment]
Thanks!

**Pull request merged**